update-orphaning/index.html

```diff
@@ -10,12 +10,6 @@
 <script type="text/javascript" src="index.js" charset="utf-8"></script>
 </head>
 <body onload="initDashboard()">
-  <div style="border: 2px solid #C13832; background-color: #D7D3C8; padding: 0.5em; color: #4D4E53; margin-bottom: 1em;">
```

Comment:
The fix linked, does that apply for data from past dates? In this case maybe we should leave a note that this issue is not longer a problem from when the PR was merged?

Reply:
It should be fine - I have restarted couple of recent runs and folks are looking only at the latest aggregation here.